package.json

```diff
@@ -17,8 +17,19 @@
     "eslint": "^9.6.0",
     "globals": "^15.8.0",
     "husky": "^9.0.11",
+    "lint-staged": "^15.2.7",
```

Comment:
why are these packages as a dev dependency and not a normal dependency?

Reply:
A normal dependency should contain only the packages that are required during the production phase, as all of these packages are only required during the development phase, adding them as a dev-dependency will reduce some load during the production phase.

Reply:
what do you mean by **_production phase_** and **_load_** during production phase

Reply:
The production phase is the phase during which the website is live; hosted on a server where it is being accessed by the users in the real time.

By load during this phase I am referring to the download time it will take and also the resources it will consume, like memory, cpu etc due to the packages downloaded.

That's why keeping only crucial dependencies for this phase is important as it will affect the speed of the website.